float32 score
float32 max_q_value
